src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -61,11 +61,19 @@
     Compute a default ServiceWorkerAssetsManifestVersion value by combining all the asset hashes.
     This is useful because then clients will only have to repopulate caches if the contents have changed.
   -->
-  <Target Name="_ComputeDefaultServiceWorkerAssetsManifestVersion" Condition="'$(ServiceWorkerAssetsManifest)' != ''">
+  <Target Name="_ComputeDefaultServiceWorkerAssetsManifestVersion"
```

Comment:
Is it possible to add a Build integration test for this?

Reply:
There already is one, in that it checks for the presence of a hash. The trouble was the hash was useless before because it was the hash of an empty string.

I'm not sure there's a good way to verify the hash is "correct" except by reproducing all the logic involved in computing it, some of which is undefined (e.g., the ordering of items in the list we use to compute the hash).

Perhaps in theory we could have an integration test that publishes once, modifies the source, publishes again, and checks the hash has changed. However that's really specialised and intricate and we don't tend to have such complex tests for other aspects of the build, such as incrementalism.

My hope is the assertion I've added is the most practical way to ensure we're not in this exact situation again.